mlflow/metrics/genai/__init__.py

```diff
@@ -0,0 +1,2 @@
+from mlflow.metrics.genai.base import EvaluationExample
+from mlflow.metrics.genai.metric_definitions import * 
```

Comment:
Needed for https://github.com/mlflow/mlflow/pull/10138/files#diff-8f6c5bf82b3cd8071b378ed3d54f25ae369c17319ce4e1d4fd95861de9fe3b2aR588